WordPress/src/main/res/layout/site_creation_preview_web_view_container.xml

```diff
@@ -42,7 +42,7 @@
             </FrameLayout>
         </com.google.android.material.card.MaterialCardView>
 
-        <org.wordpress.android.ui.WPWebView
```

Comment:
@malinajirka Here's the change for the scrolling. I'll be honest I didn't do a deep dive into this, I just saw that `NestedWevView` is what we're using here: https://github.com/wordpress-mobile/WordPress-Android/blob/95b15937bb6a342f3cdfb80e8e5e744a0c0aff82/WordPress/src/main/res/layout/home_page_picker_preview_fragment.xml#L44 and it seems like this has the same nested view hierarchy problem mentioned here https://github.com/wordpress-mobile/WordPress-Android/blob/95b15937bb6a342f3cdfb80e8e5e744a0c0aff82/WordPress/src/main/java/org/wordpress/android/widgets/NestedWebView.kt#L14-L22

Basically just calling this out incase your more familiar with this class.

Reply:
I see, nice catch! This bug was most probably introduced when we re-themed/designed AppBar on all screens of the app.